src/psyclone/psyGen.py

```diff
@@ -253,6 +254,11 @@ def create(self, invoke_info):
         elif self._type == "gocean1.0":
             from psyclone.gocean1p0 import GOPSy
             return GOPSy(invoke_info)
+        elif self._type == "nemo":
```

Comment:
Please add argumet information to the docstring in the parent method.

Reply:
Done.